test.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# ./test.sh                bring up, make database, test, bring down
+# for development:
+# ./test.sh -u             bring up background and load database if needed
+# ./test.sh [params]       run tests, passing optional params to inner test
+# ./test.sh -s             stop test containers without removing
+# ./test.sh -d             clean test containers
+
+COMPOSE_FILE_LOC=docker/docker-compose.test.yml
+COMPOSE_PROJECT_NAME_ORIGINAL=listenbrainz_test
+
+# Project name is sanitized by Compose, so we need to do the same thing.
+# See https://github.com/docker/compose/issues/2119.
+COMPOSE_PROJECT_NAME=$(echo $COMPOSE_PROJECT_NAME_ORIGINAL | awk '{print tolower($0)}' | sed 's/[^a-z0-9]*//g')
+TEST_CONTAINER_NAME=listenbrainz
+TEST_CONTAINER_REF="${COMPOSE_PROJECT_NAME}_${TEST_CONTAINER_NAME}_1"
+
+#docker-compose -f $COMPOSE_FILE_LOC -p $COMPOSE_PROJECT_NAME build
+#    docker ps -a --no-trunc  | grep $COMPOSE_PROJECT_NAME \
+#        | awk '{print $1}' | xargs -r --no-run-if-empty docker stop
+#    docker ps -a --no-trunc  | grep $COMPOSE_PROJECT_NAME \
+#        | awk '{print $1}' | xargs -r --no-run-if-empty docker rm
+
+
+if [[ ! -d "docker" ]]; then
+    echo "This script must be run from the top level directory of the listenbrainz-server source."
+    exit -1
+fi
+
+
+function bring_up_db {
+    docker-compose -f $COMPOSE_FILE_LOC -p $COMPOSE_PROJECT_NAME up -d db influx redis
```

Comment:
true, perhaps not needed. I originally had it because I did
```
docker-compose up <databases>
docker-compose run listenbrainz dockerize ... <setup db>
docker-compose up -d
docker-compose logs -f listenbrainz
```

then I realised that we could skip the `up -d` and `logs` and just run py.test straight from `docker-compose run`.
In this case, if we run `docker-compose run listenbrainz dockerize ... <setup db>` then it should load the database containers first, right?